lib/ssh_scan/error.rb

```diff
@@ -0,0 +1,10 @@
+module SSHScan
+  class Error < Exception
```

Comment:
Maybe we should be more explicit here about what type of error.

What about calling it "ConnectTimeout"?